### PR TITLE
Lipinski Rule of 5 was incorrectly described.

### PR DESCRIPTION
**Pull request by KollinRT**
#### What does this implement/fix? Explain your changes.
There was an error in how Lipinski's Rule of 5 was described. I have fixed the changes and updated the documentation in docs to follow the original paper and also fixes consistency with the code example below.

#### Any other comments?
Great project and love the work!

### CODE REVIEW

File: Docs/Book/GettingStartedInPython.rst [hunk 1]
```diff
@@ -3720,9 +3720,9 @@ Lipinski Rule of 5
 Lipinski's "Rule of 5" [#lipinski]_ was introduced to estimate the oral bioavailability of molecules. Poor absorption is likely if the molecule violates more than one of the following conditions: 
 
 * Molecular Weight <= 500 Da
-* No. Hydrogen Bond Donors <= 10
-* No. Hydrogen Bond Acceptors <= 5
-* LogP <= 5
+* No. Hydrogen Bond Donors <= 5
+* No. Hydrogen Bond Acceptors <= 10
```
Comment (greglandrum) [inline Docs/Book/GettingStartedInPython.rst:3724]: Thanks for catching this swap.

File: Docs/Book/GettingStartedInPython.rst [hunk 2]
```diff
@@ -3720,9 +3720,9 @@ Lipinski Rule of 5
...
+* LogP <= 5 OR MLogP <= 4.15
```
Comment (greglandrum) [inline Docs/Book/GettingStartedInPython.rst:3725]: I don't think the addition of the "MLogP" value is correct. Where did you get that?

**Review by greglandrum — CHANGES_REQUESTED** [review]
Question about one of the corrections.

File: Docs/Book/GettingStartedInPython.rst (hunk 2 above)
Comment (KollinRT) [reply Docs/Book/GettingStartedInPython.rst:3725]: From Lipinski's 2001 paper "Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings". There is a line on page 10 that says

> The ‘rule of 5’ states that: poor absorption or permeation are more likely when:
> There are more than 5 H-bond donors (expressed as the sum of OHs and NHs);
> The MWT is over 500;
> The Log P is over 5 (or MLogP is over 4.15); 
> There are more than 10 H-bond acceptors (expressed as the sum of Ns and Os);
> Compound classes that are substrates for biological transporters are exceptions to the rule.

MLogP is just another calculation for Moriguchi method to calculate LogP -- although CLogP I believe the more accurate and less contested metric in this space.
(Moriguchi, Ikuo, et al. "Simple method of calculating octanol/water partition coefficient." Chemical and pharmaceutical bulletin 40.1 (1992): 127-130.)

Comment (greglandrum) [reply Docs/Book/GettingStartedInPython.rst:3725]: > From Lipinski's 2001 paper "Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings". There is a line on page 10 that says
> 
> > The ‘rule of 5’ states that: poor absorption or permeation are more likely when:
> > There are more than 5 H-bond donors (expressed as the sum of OHs and NHs);
> > The MWT is over 500;
> > The Log P is over 5 (or MLogP is over 4.15);
> > There are more than 10 H-bond acceptors (expressed as the sum of Ns and Os);
> > Compound classes that are substrates for biological transporters are exceptions to the rule.
> 
> MLogP is just another calculation for the Moriguchi method to calculate LogP -- although CLogP I believe the more accurate and less contested metric in this space. (Moriguchi, Ikuo, et al. "Simple method of calculating octanol/water partition coefficient." Chemical and pharmaceutical bulletin 40.1 (1992): 127-130.)

Thanks for that pointer. Given that this is definitely not the popular understanding of the Ro5, I would prefer to leave it out.

**Review by greglandrum — APPROVED** [review]
LGTM

**Comment by greglandrum** [comment]
Thanks for the contribution @KollinRT

**Pull request merged**